src/cli/commands/start.ts

```diff
@@ -102,7 +106,7 @@ Serve static content from a folder and run an API from another folder
 swa start ./output-folder --api-location ./api
 
 Use a custom command to run framework development server at startup
-swa start http://localhost:3000 --run "npm start"
+swa start http://localhost:3000 --run-build "npm start"
```

Comment:
this should be `--run` not `--run-build`

Reply:
Fixed, `--run-build` was removed in favor or `swa build`